### PR TITLE
limit order: support wrap native in input

### PR DESCRIPTION
**Pull request by nguyenhoaidanh**
https://team-kyber.atlassian.net/browse/KGP-443

### CODE REVIEW

File: src/pages/Verify/index.tsx [hunk 1]
```diff
@@ -142,8 +142,14 @@ function Verify() {
                       Your email have been verified{isVerifyExternal ? null : ' by KyberSwap.com.'}.
                     </Text>{' '}
                     If it has been more than a few days and you still haven’t receive any notification yet, please
-                    contact us through our channels. You will be redirected to{' '}
-                    {isVerifyExternal ? <ExternalLink href={redirectUrl}>{redirectUrl}</ExternalLink> : 'our Swap page'}{' '}
+                    contact us through our channels.
+                    <br />
```
Comment (nguyenhoaidanh) [inline src/pages/Verify/index.tsx:146]: only add <br/>

File: src/components/SearchModal/CurrencySearch.tsx [hunk 2]
```diff
@@ -159,40 +159,38 @@ export function CurrencySearch({
...
+    return filterTokens(chainId, commonTokens as Token[], debouncedQuery).filter(filterWrapFunc)
+  }, [commonTokens, debouncedQuery, chainId, filterWrapFunc])
 
   const filteredSortedTokens: Token[] = useMemo(() => {
+    const nativeToken = NativeCurrencies[chainId]
+    const tokensWithNative = [nativeToken, ...fetchedTokens] as Token[]
     if (!debouncedQuery) {
-      // only sort whitelist token,  token search we don't sort
-      const sorted = fetchedTokens.sort(tokenComparator)
-      const symbolMatch = debouncedQuery
```
Comment (nguyenhoaidanh) [inline src/components/SearchModal/CurrencySearch.tsx:183]: redundant logic debouncedQuery alway empty

File: src/components/Button/index.tsx [hunk 3]
```diff
@@ -264,13 +264,43 @@ export function ButtonDropdownLight({
...
+  disabled: boolean
+  loading: boolean
+  confirmed?: boolean
+  text?: ReactNode
+}) => {
+  return (
+    <BtnInfoWrapper disabled={disabled} altDisabledStyle={loading} confirmed={confirmed} onClick={onClick}>
+      <MouseoverTooltip width="300px" text={tooltipMsg} disableTooltip={loading}>
+        <Flex
+          style={{ alignItems: 'center', height: 44, paddingRight: 8, paddingLeft: 2 }}
```
Comment (viet-nv) [inline src/components/Button/index.tsx:294]: use `sx` instead. sx will generate classname, style will generate inline style

Comment (viet-nv) [reply src/components/Button/index.tsx:294]: btw, Flex support almost css props
<Flex alignItems="center" height="44px" ...>

File: src/components/SearchModal/CurrencySearch.tsx [hunk 4]
```diff
@@ -289,27 +283,21 @@ export function CurrencySearch({
...
+        addressesToFetch.push(address)
       })
-      if (promises.length) {
-        const data = await Promise.allSettled(promises)
-        data.forEach(el => {
-          if (el.status !== 'fulfilled') return
-          const tokenResponse = el.value
-          if (tokenResponse) result.push(tokenResponse)
-        })
+      if (addressesToFetch.length) {
```
Comment (nguyenhoaidanh) [inline src/components/SearchModal/CurrencySearch.tsx:298]: call 1 api instead

File: src/components/Button/index.tsx (hunk 3 above)
Comment (nguyenhoaidanh) [reply src/components/Button/index.tsx]: updated.

**Review by viet-nv — APPROVED** [review]

**Pull request merged**